src/tests/JIT/Regression/JitBlue/GitHub_85129/GitHub_85129.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using Xunit;
+
+public class Program
+{
+    [Fact]
+    public static int TestEntryPoint()
+    {
+
+        Vector256<int> v256Shuffle = Vector256.Create(100, 101, 102, 103, 104, 105, 106, 107);
+        Vector256<int> v256ShuffleExpectedResult = Vector256.Create(107, 105, 0, 101, 106, 104, 0, 100);
+        Vector256<int> v256ShuffleActualResult = Vector256Shuffle(v256Shuffle);
+        if(v256ShuffleExpectedResult != v256ShuffleActualResult)
+        {
+            return 1;
```

Comment:
I was unfamiliar with the `int` return values for this type of test. Looking at a couple other tests, I'm seeing:

```c#
        const int Pass = 100;
        const int Fail = -1;
```

1. Is `1` correct here, or should it be `-1`?
2. I suggest defining the consts to help other readers/reviewers unfamiliar with this pattern